Leojr2021/impacto-whatsapp-button
# feat: create the whatsapp button

#### What problem is this solving?
Create the WhatsApp icon for my ecommerce.

#### How to test it?

Click to the WhatsApp icon, then I should redirect to send  normal WhatsApp message.


#### Screenshots or example usage:

![image](https://github.com/Leojr2021/impacto-whatsapp-button/assets/78285858/68e3fe8a-a38d-4b2f-9261-bb61bd81685d)
![image](https://github.com/Leojr2021/impacto-whatsapp-button/assets/78285858/7d0f81e6-cebc-4202-91ed-fd8408ad32c4)
